simplq/src/components/common/Loading/Loading.jsx

```diff
@@ -2,22 +2,22 @@ import React from 'react';
 import PropagateLoader from 'react-spinners/PropagateLoader';
 import styles from './Loading.module.scss';
 
-export default ({ children, actionStatus }) => {
-  if (!actionStatus || actionStatus === 'pending') {
-    return (
-      <div className={styles.main}>
-        <PropagateLoader color="#3a3768" />
-      </div>
-    );
+export default ({ children, actionStatus, isLoading }) => {
+  if (actionStatus === 'fulfilled' || isLoading === false) {
+    return <>{children}</>;
   }
 
-  if (!actionStatus || actionStatus === 'rejected') {
+  if (actionStatus === 'rejected') {
     return (
       <div className={styles.main}>
         An unknown error occured. Please look at the console log for more nfo.
       </div>
     );
   }
 
-  return <>{children}</>;
+  return (
```

Comment:
There is an edge case here: If actionStatus and isLoading are undefined, it will spin for ever. 

Is that a bug or a feature 😝  (I don't think so)

Reply:
It is a feature if you want to use a spinner as before. Although, I was considering making at least one param required. And when I think about it, I have to add PropTypes to warn if params are not proper.

Reply:
Yes. If we are using as before in this context, we are not using it ideally.